<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCM_WIN_StarterList</class>
 <widget class="QMainWindow" name="NCM_WIN_StarterList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="4">
     <layout class="QGridLayout" name="gridLayout_StarterList"/>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label_Name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;</string>
      </property>
      <property name="text">
       <string>Stage name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_End">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;</string>
      </property>
      <property name="text">
       <string>--:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Approximate stage end</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QPushButton" name="pushButton_DE">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3" rowspan="2">
     <widget class="QPushButton" name="pushButton_BTN">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 35px</string>
      </property>
      <property name="text">
       <string>üêªüêØüê±‚Äçüë§</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_Time">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;</string>
      </property>
      <property name="text">
       <string>--:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Current Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionget_data"/>
   </widget>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionget_data">
   <property name="text">
    <string>get data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
